site/content/docs/browser-checks/_index.md

```diff
@@ -25,135 +26,119 @@ Your critical interactions might be:
-
-if (response.status() > 399) {
-  throw new Error(`Failed with response code ${response.status()}`)
-}
-
-await browser.close()
+{{< tab "TypeScript" >}}
+ ```ts
+import { test } from '@playwright/test'
+
+test('Visit Checkly HQ page', async ({ page }) => {
+  const response = await page.goto('https://checklyhq.com')
+
+  // Test that the response did not fail
+  if (response.status() > 399) {
+    throw new Error(`Failed with response code ${response.status()}`)
+  }
+})
  ```
 {{< /tab >}}
 {{< /tabs >}}
 
 {{< info >}}
-Checkly currently supports using only **Chromium** with Playwright and Puppeteer.
+Checkly currently supports only using **Chromium** with Playwright Test and Playwright library.
 {{< /info >}}
 
-{{< warning >}}
-Note: It is not possible to use both Playwright and Puppeteer in the same Browser Check script.
-{{< /warning >}}
-
-
 ## Breaking down a Browser check step-by-step
 
-Let's look at a more real life example and break down each step. The code below logs into Checkly, waits for the dashboard
-to fully load and then snaps a screenshot.
+Let's look at a breakdown of a real-life scenario. The code below logs into Checkly, and waits for the dashboard to fully load.
 
 {{< tabs "Breakdown example" >}}
-{{< tab "Playwright" >}}
+{{< tab "JavaScript" >}}
 ```js
-const playwright = require('playwright') // 1
-const browser = await playwright.chromium.launch()
-const page = await browser.newPage()
+const { test } = require('@playwright/test') // 1
 
-await page.goto('https://app.checklyhq.com/login') // 2
+test('Login to Checkly', async ({ page }) => {
+  await page.goto('https://app.checklyhq.com/login') // 2
 
-await page.type('input[type="email"]', 'john@example.com') // 3
-await page.type('input[type="password"]','mypassword')
-await page.click('.btn.btn-success.btn-block')
+  await page.fill('input[type="email"]', 'john@example.com') // 3
+  await page.fill('input[type="password"]', 'mypassword')
+  await page.getByRole('button', { name: 'Log In' }).click() // 4
 
-await page.waitForSelector('.status-table') // 4
-await page.screenshot({ path: 'checkly_dashboard.png' })
-await browser.close()
+  await page.getByTestId('home-dashboard-table').isVisible() // 5
+})
  ```
 {{< /tab >}}
-{{< tab "Puppeteer" >}}
-```js
-const puppeteer = require('puppeteer') // 1
-const browser = await puppeteer.launch()
-const page = await browser.newPage()
+{{< tab "TypeScript" >}}
+```ts
+import { test } from '@playwright/test' // 1
 
-await page.goto('https://app.checklyhq.com/login') // 2
+test('Login to Checkly', async ({ page }) => {
+  await page.goto('https://app.checklyhq.com/login') // 2
 
-await page.type('input[type="email"]', 'john@example.com') // 3
-await page.type('input[type="password"]','mypassword')
-await page.click('.btn.btn-success.btn-block')
+  await page.fill('input[type="email"]', 'john@example.com') // 3
+  await page.fill('input[type="password"]', 'mypassword')
```

Comment:
I think `type` should be preferred to `fill` since it emits multiple keyboard events.